src/data/balance/tms.ts

```diff
@@ -1,38 +1,38 @@
 import { ModifierTier } from "#enums/modifier-tier";
-import { Moves } from "#enums/moves";
+import { MoveId } from "#enums/moves";
 import { Species } from "#enums/species";
 
 interface TmSpecies {
   [key: number]: Array<Species | Array<Species | string>>;
 }
 
-export const reverseCompatibleTms: Moves[] = []; /*[
-    Moves.TAKE_DOWN,
-    Moves.TOXIC,
-    Moves.RAGE,
-    Moves.MIMIC,
-    Moves.DOUBLE_TEAM,
-    Moves.BIDE,
-    Moves.REST,
-    Moves.SUBSTITUTE,
-    Moves.SNORE,
-    Moves.PROTECT,
-    Moves.ENDURE,
-    Moves.SWAGGER,
-    Moves.ATTRACT,
-    Moves.SLEEP_TALK,
-    Moves.RETURN,
-    Moves.FRUSTRATION,
-    Moves.HIDDEN_POWER,
-    Moves.FACADE,
-    Moves.SECRET_POWER,
-    Moves.NATURAL_GIFT,
-    Moves.CAPTIVATE,
-    Moves.ROUND
+export const reverseCompatibleTms: MoveId[] = []; /*[
+    MoveIdTAKE_DOWN,
```

Comment:
```suggestion
    MoveId.TAKE_DOWN,
```
(and so on for the rest of the "fixes")

Reply:
f...
Thanks good catch. Early bird doesn't always catch the worm

Reply:
Fixed: https://github.com/Despair-Games/poketernity/pull/402/commits/19aaa07c5c2b632960272e6e745cbc7f4c0db769